### ISSUE TITLE
GMCP aktivieren

### ISSUE THREAD

**Issue by Kebap** [Mundron/GUI-fuers-MG#2]
Wenn ich dein Paket mit einem neuen Char benutzen will, klappt das nicht automatisch.
Hintergrund ist, dass MG normalerweise erstmal nur die "Char" Daten per gmcp sendet.
Dein Paket erwartet aber die "MG.char" Daten. Diese werden aber nicht angefordert.

Das lässt sich bspw. so erreichen:
`sendGMCP( [[Core.Supports.Set [ "MG.char 1", "MG.room 1", "comm.channel 1" ] ]])`

**Comment by Mundron** [comment]
Ja, alle meine Skripte brauchen das und daher hab ich es erstmal ausgeklammert um es nicht in jedem Paket drin zu haben. Wobei das vermutlich nicht stören wird. Ich werde es bei Gelegenheit hinzufügen. Danke!
